executor/include/executor/trait/common_traits.hpp

```diff
@@ -38,4 +55,78 @@ struct equality_comparable<
 template <typename T1, typename T2>
 constexpr bool equality_comparable_v = equality_comparable<T1, T2>::value;
 
+/**
+ *   Custom Traits
+ **/
+
+// is_instance_of_base
+namespace detail {
+
+template <typename Executor, template <typename...> class Type>
+struct is_instance_of_any_impl : std::false_type {};
+
+template <template <typename...> class Type, typename... Args>
+struct is_instance_of_any_impl<Type<Args...>, Type> : std::true_type {};
```

Comment:
`is_instance_of`?

Reply:
Not quite sure I understood what you meant.